src/SFA.DAS.AssessorService.Web/Validators/Standard/OptInStandardVersionViewModelValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.AssessorService.Api.Types.Models.CompaniesHouse;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Helpers;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+using StructureMap.Query;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class OptInStandardVersionViewModelValidator : AbstractValidator<OptInStandardVersionViewModel>
+    {
+        private string _standardVersion;
```

Comment:
See comment below on the OptOut validator for better design

Reply:
Done